mage_ai/frontend/package.json

```diff
@@ -4,9 +4,9 @@
   "private": true,
   "scripts": {
     "build": "next build",
-    "export": "next build && next export",
-    "export_prod": "next build && next export -o ../server/frontend_dist",
-    "export_prod_base_path": "next build && next export -o ../server/frontend_dist_base_path_template",
+    "export": "next build",
+    "export_prod": "next build && mv out ../server/frontend_dist",
```

Comment:
When I tested exporting the frontend build, it just moved an `out` folder with the new build inside of the `frontend_dist` directory, but we need the build to be inside the `frontend_dist` directory itself.  Not sure if there's a better way to go about this, but this command had worked for me (requires adding a `distDir` prop in the next.config.js file):
`"export_prod": "next build && rm -rf ../server/frontend_dist && mv frontend_dist ../server",`

Reply:
addressed in 8d673da06